### ISSUE TITLE
v2: Errors should be reported by default

### ISSUE THREAD

**Issue by azeemba** [azeemba/eslint-plugin-json#27]
I was testing the latest version after I made the v2 release and it seems that if the `json/json` rule is not explicitly set to error in .eslintrc, then no errors are being reported.

@AdrieanKhisbe do you think this should be the default behavior? I think if the extension is enabled, then we should report the errors by default. Users can disable the rules using the ruleIds if they want.

I have unpublished the v2 release for now.

**Comment by AdrieanKhisbe** [comment]
Hey @azeemba,
I made it on purpose, I thinks it's better a plugin does not activate anything by itself.

What we can do instead is ship a config (or two) to easily activate it

```json
{
   "extends": "plugin:json/some-name-to-find",
   "plugins": ["json"]
}
```
with some one named `with-comments`, and one other `strict`.

What do you think? :)

But anything, we indeed have to communicate it clearly to the users migrating to 2.0

**Comment by azeemba** [comment]
Since JSON is defined pretty strictly, I would have expected the default behavior to be "strict". And we can provide a config for "with-comments" for users that want to do that.

However, I have not been doing a lot of JS recently so I haven't kept up with what behavior most plugins follow. If you believe it is better for the plugin to not enable anything by default, then we can go with that.

**Comment by Cherry** [comment]
I'd definitely vote for a "recommended" ruleset users can `extend` from in their ESLint config, which would essentially be strict JSON. Users can then override specific rules (like comments) if they need those enabled. This is how [`eslint-plugin-node`](https://www.npmjs.com/package/eslint-plugin-node) does it, for example; nothing is enabled by default until you extend from `plugin:node/recommended` or enable specific rules.

Migration details should definitely be explicit in the README though, as this is pretty massive change from how this plugin used to function.

**Comment by AdrieanKhisbe** [comment]
`recommended` is indeed better than `strict` / `base` / `whatever`.

Will try to do add the two config, and the Migration details in the hours to come
Keeping you posted
